docs: Add troubleshooting for disabled alerts still visible on nodes (#22721)
netdata/netdata: docs/netdata-agent/configuration/dynamic-configuration.md

1. **Check your assigned role**: Go to **Space Settings → Users** and verify your role. If you are not an Admin or Manager, ask a Space Admin to upgrade your role. See the [Role-Based Access Model documentation](/docs/netdata-cloud/authentication-and-authorization/role-based-access-model.md) for the full permissions table.
2. **Verify your subscription plan**: If you are on the Community plan, [upgrade to a paid plan](https://www.netdata.cloud/pricing/) or ask a Space Admin to do so. A paid plan is required for all Dynamic Configuration actions except **List All**.

### Disabled Alert Template Still Appears on Nodes

> If you disabled an alert template (for example `10min_cpu_usage`) through the Dynamic Configuration Manager but still see it firing on an individual node, the disable only applies to the node you applied it to. **Each node — including every child in a parent/child streaming setup — evaluates its own alerts from its own health configuration.** Disabling an alert template on one node does not change any other node's configuration.

Reviewer: ```suggestion
The Dynamic Configuration Manager has a function to bulk-apply configurations to multiple nodes. Without it, any change you do is for the specific node you have selected. So, if you want to disable an alert template (for example `10min_cpu_usage`) for X desired nodes, you need to target all of them.
```

> **Resolution:**

Reviewer: ```suggestion
**Solution:**
```

Apply the disable to each node that is actually raising the alert. The recommended approach is to use the [Multi-Node Deployment](#multi-node-deployment) feature to select every node still raising the alert and push the disabled configuration to all of them at once.

<details>
<summary>Alternative: configure each node manually (optional)</summary>

> If you prefer not to use Multi-Node Deployment, disable the alert directly in each affected node's local health configuration:

Reviewer: ```suggestion
If you prefer not to use Multi-Node Deployment, disable the alert directly in each desired node's local health configuration:
```
